src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+  public static var coreYellow1000: Color {
+    return Color.init("coreYellow1000", bundle: bundle)
+  }
+}
+```
+
+
+----
+
+
+## Bonus: high contrast mode
+
+<ThemedImage style={{maxWidth: '15rem', float: 'left', marginRight: 'var(--size-padding-base)', marginBottom: 'var(--size-padding-base)'}}>
+
+![iOS accessibility settings showing high contrast](images/ios-hc-light.png)
+![iOS accessibility settings showing high contrast](images/ios-hc-dark.png)
+
+</ThemedImage>
+
+iOS has a setting called "increase contrast" in its accessibility menu. This is separate from light and dark mode so technically you can have 4 color modes: light, light high contrast, dark, and dark high contrast.
+
+<div className="clear" />
+
+On the web there is a newer media query called `prefers-contrast` which can be set to: `no-preference`, `more`, and `less`. This media query is in the [media queries 5 draft spec](https://drafts.csswg.org/mediaqueries-5/#descdef-media-prefers-contrast). Here is an [MDN article on prefers-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast) as well.
+
+The browser compatibility table as of this writing is all no support except Firefox with a special flag. The macOS 'increase contrast' accessibility setting does not yet work with any browser. 
+
+Android has an accessibility setting called 'high contrast text', which is an experimental setting at the time of this writing. As far as I'm aware, this setting makes all text pure black or pure white based on its background color. An app developer doesn't have access to changing colors based on this setting though, I don't think.
+
+Overall support for high contrast mode is not great across platforms, but we can still start to build it into our design tokens. Platforms that support it can get it, and others won't. It is a good idea to plan for the future in a progressive enhancement way, especially as it relates to accessibility.
+
+Both approaches outlined would work for adding more color modes. The single token approach would look like this:
+
+```json
+{
+  "color": {
+    "background": {
+      "primary": {
+        "value": "{color.core.neutral.0.value}",
+        "darkValue": "{color.core.neutral.900.value}",
+        "hcValue": "{color.core.neutral.0.value}",
+        "hcdarkVlue": "{color.core.neutral.1000.value}"
+      },
+      //...
+```
+
+Custom format and action code would need to be adjusted accordingly because they reference keys like `darkValue`.
+
+The multi-file approach would either add `hc-tokens/` directory or add `.hc.json` file extension. 
+
+
+----
+
+
+## Conclusion
+
+If you made it this far, congratulations! Hopefully I didn't scare you off you can take this and apply it to your design token setup. Even though this is a long article, there is a lot more information in the [repository](repo) which includes heavily commented code and instructions. 
```

Comment:
Hopefully I didn't scare you off you can > Hopefully, I didn't scare you off. You can